mobile/components/Messages/Author.js

```diff
@@ -17,13 +22,19 @@ const AuthorWrapper = styled.View`
   align-items: flex-end;
 `;
 
-export default ({ author, me }: Props) => {
+const Byline = ({ author, me, navigation }: Props) => {
   return (
-    <AuthorWrapper me={me}>
-      {!me && <Avatar src={author.user.profilePhoto} size={16} radius={8} />}
-      <Text type="footnote" color={props => props.theme.text.alt}>
-        <Text bold>{author.user.name}</Text> (@{author.user.username})
-      </Text>
-    </AuthorWrapper>
+    <TouchableHighlight
+      onPress={() => navigation.navigate(`User`, { id: author.user.id })}
```

Comment:
Same thing here as with the Avatar—I can already think of a case where we don't want this component clickable (message replies), so let's move this a level up out, of this component, and keep it focused on its actual use.